src/Init/NotationExtra.lean

```diff
@@ -357,4 +357,27 @@ meta def insertUnexpander : Lean.PrettyPrinter.Unexpander
   | `($_ $a { $ts:term,* }) => `({$a:term, $ts,*})
   | _ => throw ()
 
+namespace Parser.Command
+
+/-- An attribute instance with optional scoped/local modifier. -/
+syntax scopedNSAttrInstance := attrKind attr
+
+/-- Attributes syntax `@[...]` for scoped namespace commands. -/
+syntax scopedNSAttrs := "@[" sepBy1(scopedNSAttrInstance, ", ") "] "
```

Comment:
@Kha, can you advise here? I should be using `` `Lean.Parser.Term.attributes`` here, but that it only defined much later. This feels really hacky, and then requires the even hackier `setKind` in `private def toTermAttrs` below.